polars/polars-core/src/chunked_array/ops/mod.rs

```diff
@@ -373,7 +373,7 @@ pub trait ChunkSet<'a, A, B> {
     ///
     /// ```rust
     /// # use polars_core::prelude::*;
-    /// let ca = UInt32Chunked::new_from_slice("a", &[1, 2, 3]);
+    /// let ca = Int32Chunked::new_from_slice("a", &[1, 2, 3]);
```

Comment:
arrow2 already supports taking `UInt32`, so we could also revert this change in polars, so that the API is the same.

Reply:
I think the failure of this docstrings was a proof that this branch has some traits better defined.

It failed because we cannot change the values in a `UIn32tArray` to a negative value. Which seems correct to me, so I changed the array to a `Int32Array`. Note that this array is not used as index array, in this example we use a `Vec<usize>` for that.

Reply:
ah, got it. Good catch :)